specs/DisableNavigatingBackAndForward.md

```diff
@@ -73,25 +73,36 @@ void WebView_NavigationStarting(object sender, CoreWebView2NavigationStartingEve
 // Enums and structs
 [v1_enum]
 typedef enum COREWEBVIEW2_NAVIGATION_KIND {
+  /// Default kind to cover all unknown kind.
+  /// 
+  /// Note: This field should not be explicitly used by developers, because this kind contains
+  /// little valid information and may change in future.
+  COREWEBVIEW2_NAVIGATION_KIND_UNKNOWN,
+
   /// A navigation caused by CoreWebView2.Reload(), location.reload(), the end user 
   /// using F5 or other UX, or other reload mechanisms to reload the current document 
   /// without modifying the navigation history.
   COREWEBVIEW2_NAVIGATION_KIND_RELOAD,
+
   /// A navigation back or forward to a different entry in the session navigation history. 
   /// For example via CoreWebView2.Back(), location.back(), the end user pressing Alt+Left 
   /// or other UX, or other mechanisms to navigate forward or backward in the current 
   /// session navigation history.
+  ///
+  /// Note: This doesn't distinguish forward from backward navigations because we can't 
+  /// distinguish it from origin source `NavigationType`
   COREWEBVIEW2_NAVIGATION_KIND_BACK_OR_FORWARD,
-  /// A navigation to a different document. This can be caused by CoreWebView2.Navigate(), 
+
+  /// A navigation to another document. This can be caused by CoreWebView2.Navigate(), 
   /// window.location.href = '...', or other WebView2 or DOM APIs that navigate to a specific URI.
-  COREWEBVIEW2_NAVIGATION_KIND_DIFFERENT_DOCUMENT,
+  COREWEBVIEW2_NAVIGATION_KIND_NEW_DOCUMENT,
+
 } COREWEBVIEW2_NAVIGATION_KIND;
 
 /// Extend `NavigationStartingEventArgs` by adding more information.
 [uuid(39A27807-2365-470B-AF28-885502121049), object, pointer_default(unique)]
-interface ICoreWebView2NavigationStartingEventArgs3 : ICoreWebView2NavigationStartingEventArgs2 {
-
-  /// Indicates if this navigation is reload, back/forward or navigating to a different document 
+interface ICoreWebView2ExperimentalNavigationStartingEventArgs2 : IUnknown {
```

Comment:
We use the expected actual names in the doc not the experimental names

```suggestion
interface ICoreWebView2NavigationStartingEventArgs3 : ICoreWebView2NavigationStartingEventArgs2 {
```

Reply:
Will do.